migrations/2025-04-25-090000_trusted-publishing/up.sql

```diff
@@ -0,0 +1,64 @@
+create table github_oidc_configs
```

Comment:
should this be names `oidc_github_configs` instead so that all the related tables use a `oidc_` prefix?

Reply:
Since this would only be used for a trusted publishing service, should we add trusted publishing (or its abbreviation/alias) to the table name? This is considering we might have other services that also have their own OIDC-related configurations. However, we could also rename it later when we actually need to, though.

Reply:
I've avoided `trusted_publishing_` so far because it seems quite long, and for `tp_` it isn't really obvious what it stands for, but I'm open to suggestions :)

Reply:
maybe we could add `_publisher` after the provider (e.g., `github_publisher`), similar to how PyPI has the `_publishers` suffix.

Reply:
> should this be names `oidc_github_configs` instead so that all the related tables use a `oidc_` prefix?

I think that makes sense.

Reply:
> I've avoided `trusted_publishing_` so far because it seems quite long, and for `tp_` it isn't really obvious what it stands for, but I'm open to suggestions :)

I just came up with `trustpub_`. Short and somewhat self-describing with enough context. I think I prefer that to `oidc_`.

Reply:
I changed the names to `trustpub_configs_github`, `trustpub_tokens` and `trustpub_used_jtis`